convert/convert/config.py

```diff
@@ -4,14 +4,15 @@
 
 import boto3
 from botocore.client import BaseClient
-from convert import logger
 from dotenv import load_dotenv
 from mypy_extensions import KwArg, VarArg
 from pydantic import BaseSettings, Field
 from requests import Session
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
 
+from convert import logger
```

Comment:
Here you import logger from another place.
But in the code down below you create several loggers in different modules inplace.

I think it should one way or another - create one logger instance and import it everywhere we need it or create different instances in every module if that's needed.
The first approach seems more clean